api/proto/evecommon/devmodelcommon.proto

```diff
@@ -19,6 +19,10 @@ enum PhyIoType {
   PhyIoNetWLAN = 5;
   PhyIoNetWWAN = 6;
   PhyIoHDMI = 7;
+  PhyIoLTE = 8;
```

Comment:
I believe that is to be in sync with controller api especially the enum number. Alternatively I can add a comment to just reserve the number.

Reply:
It would be better to just mark it as reserved. FWIW the name "LTE" as used in the Zededa controller is misleading since it captures the SIMs and modems, which typically apply from 3G, LTE, 5G, etc.

Reply:
Ok I will make the change to reserve the enum.